packages/forms/resources/views/components/checkbox-list.blade.php

```diff
@@ -157,9 +165,6 @@ class="filament-forms-checkbox-list-component-option-label flex items-center spa
                         @endif
                     >
                         <input
-                            @if ($isBulkToggleable())
```

Comment:
We still need this I think, so if the user toggles all manually it shows "Deselect all"

Reply:
I have tested it, since we do `this.checkIfAllCheckboxesAreChecked()` we do not need it anymore on the input.

Reply:
If you individually select each checkbox instead of "Select all", does it switch to "Deselect all" when you check the last one? Because I dont understand when that check would happen, `toggleAllCheckboxes` isnt run in this case

Reply:
yes it does, because of the `$nextTick()`

Reply:
but init() is only run once, not after every checkbox is individually checked, right? 🤔 sorry for pushing back on this

Reply:
you are right!... well i checked and it works fine... i got nothing 🙃
![weird](https://github.com/filamentphp/filament/assets/10007504/64d8a7f1-1e6e-4e9a-b074-4dbc5f090408)